css/store.css

```diff
@@ -0,0 +1,273 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  margin: 0;
+}
+
+/*header */
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 3rem;
+  margin-bottom: 1rem;
+  padding-top: 1rem;
+}
+
+.header-logo {
+  display: flex;
+  max-height: 50px;
+  width: 100px;
+  margin-left: 15px;
+}
+
+/* Navigation */
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin-right: 80px;
+}
+
+.nav-item {
+  padding: 0.5rem 1rem;
+}
+
+#active-nav {
+  color: rgb(116, 113, 113);
+  font-weight: 600;
+}
+.nav-link {
+  color: rgb(160, 155, 155);
+  text-decoration: none;
+  font-size: 1.4rem;
+}
+
+.nav-link:hover {
+  color: #e0633a;
+  cursor: pointer;
+}
+
+/* page two */
+/* Get today section */
+
+main {
+  display: flex;
+  width: 1440px;
+  height: 820px;
```

Comment:
Try not to set the height and width of the main section, while 1440px is a standard width for a screen there will be lots of people who are using different sizes, so try not to restrict it, if you want to set the size of the screen, try using vh/vw or %

Reply:
Figma bad influence, it's changed to % now